argocd check がこける

=== Validating Kubernetes manifests ===
  Validating k3s/apps/application/hello-world...
  Validating k3s/apps/application/dawarich/cluster...
stdin - Cluster dawarich-postgres failed validation: could not find schema for Cluster
make: *** [Makefile:35: validate-manifests] Error 1
Error: Process completed with exit code 2.

なおして。あと、原因を報告して

--- a/argocd/Makefile
+++ b/argocd/Makefile
@@ -35,8 +35,9 @@ validate-manifests: ## Validate ArgoCD Application manifests with kubeconform
 	@for dir in $(shell find $(APPS_DIR) -name 'kustomization.yaml' -exec dirname {} \;); do \
 		echo "  Validating $$dir..."; \
 		kustomize build $$dir | kubeconform \
+			-schema-location 'default' \
 			-schema-location 'https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/{{.NormalizedKubernetesVersion}}-standalone{{.KindSuffix}}/{{.ResourceKind}}.json' \
-			-skip 'Application,IPAddressPool,L2Advertisement,SealedSecret,ConfigMap,Ingress' \
+			-skip 'Application,Cluster,IPAddressPool,L2Advertisement,SealedSecret,ConfigMap,Ingress' \
 			-strict || exit 1; \
 	done
 	@echo "  All manifests validated!"

